src/test/java/pl.put.poznan.builder/ElementTest.java

```diff
@@ -0,0 +1,33 @@
+
+
+import org.junit.Test;
+import pl.put.poznan.builder.logic.Element;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+class ElementTest {
+
+    @Test
+    public void addNodeTest() {
+        Element child = new Element();
+        child.setTag("meta");
+        Element parent = new Element();
+        parent.setTag("head");
+        parent.addNode(child);
+        assertEquals("meta", parent.getChildTag());
+
+    }
+    @Test
+    public void addNode2Test() {
+        Element child = new Element();
+        child.setTag("meta");
+        Element parent = new Element();
+        parent.setTag("head");
+        parent.addNode(child);
+        assertNotEquals("head", parent.getChildTag());
+
+    }
+
+
```

Comment:
I think it would be nice to have some tests for `Element.toString()` method, it's pretty much the most important one.